src/lein_monolith/task/each.clj

```diff
@@ -33,7 +33,8 @@
       [:endure])
     (when (:subtree opts)
       [:subtree])
-    (when-let [threads (ffirst (:parallel opts))]
+    (when-let [threads (or (ffirst (:parallel opts))
+                           (System/getProperty "MONOLITH_CPU_COUNT"))]
```

Comment:
Have you tested this? The way that the arg parsing  works, `:parallel` always expects the next arg to be its argument. I haven't added support for optional arg values yet, so you'd have to implement that too.